src/pages/workspaces/workspace/Data.js

```diff
@@ -383,23 +391,42 @@ const ToolDrawer = withModalDrawer()(({ workspace, onDismiss, onIgvSuccess, sele
         div({ style: Style.modalDrawer.content }, [
           div({ style: { margin: '1rem 0' } }, [
             h(ModalToolButton, {
-              onClick: () => setToolMode('IGV'),
-              tooltip: 'Open with Integrative Genomics Viewer'
+              onClick: () => entityKey !== 'cohort' && setToolMode('IGV'),
```

Comment:
If you define `const isCohort = entityKey === 'cohort'` somewhere above, you can avoid repeating this check each time.

Also, if this is disabled, you don't need to remove the onClick manually.

(These apply to the buttons below as well)

Reply:
Done. I also got rid of `entitiesType` on line 367 since `entityKey` can be used instead.